src/styles/blocks/card.scss

```diff
@@ -0,0 +1,96 @@
+.card {
+  display: flex;
+  width: 200px;
+  flex-direction: column;
+  background-color: $white-color;
+  box-sizing: border-box;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+  transition: transform 300ms;
+
+  @include hover(transform, scale(1.2));
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px 19px 0;
```

Comment:
It's a good practice to avoid using magic numbers in your code. The values '32px', '19px' seem to be arbitrary. Consider using variables or calculations to make it clear where these numbers come from.